install_scripts/templates/kubernetes/update-apiserver-certs.sh

```diff
@@ -0,0 +1,81 @@
+#!/bin/bash
+
+set -e
+
+PROXY_ADDRESS=
+PUBLIC_ADDRESS=
+PRIVATE_ADDRESS=
+LOAD_BALANCER_ADDRESS=
+LOAD_BALANCER_PORT=
+ADDITIONAL_SANS=
+
+{% include 'common/common.sh' %}
+{% include 'common/prompt.sh' %}
+{% include 'common/log.sh' %}
+{% include 'common/system.sh' %}
+{% include 'common/selinux.sh' %}
+{% include 'common/ip-address.sh' %}
+{% include 'common/docker.sh' %}
+{% include 'common/cli-script.sh' %}
+{% include 'common/kubernetes.sh' %}
+{% include 'common/kubernetes-upgrade.sh' %}
+
+outro() {
+    printf "\n"
+    printf "\t\t${GREEN}Update${NC}\n"
+    printf "\t\t${GREEN}  Complete ✔${NC}\n"
+    printf "\n"
+}
+
+################################################################################
+# Execution starts here
+################################################################################
+
+export DEBIAN_FRONTEND=noninteractive
```

Comment:
There aren't any deb operations in this script so this can be removed. Around here would be a good place to check for root.

Reply:
cant hurt. added require root user